lib/deps/docs/preprocessAttachments.js

```diff
@@ -20,7 +20,7 @@ function preprocessAttachments(docInfos, blobType, callback) {
       return base64.atob(data);
     } catch (e) {
       var err = errors.error(errors.BAD_ARG,
-        'Attachments need to be base64 encoded');
+        'Attachment is not a valid base64 string');
```

Comment:
This is the only change I haven't been able to add a test for. I can't break any of the `npm test` tests from within this file, and the whole file seems to be ignored for Istanbul... Are there any tests for this?